src/components/response/RadioInput.tsx

```diff
@@ -22,8 +22,10 @@ export default function RadioInput({
         withAsterisk={required}
         size={'md'}
         {...answer}
+        // This overrides the answers error. Which..is bad?
+        error={answer.value && requiredValue && requiredValue.toString() !== answer.value.toString() ?  `Please select ${requiredLabel ? requiredLabel : options.find((opt) => opt.value === requiredValue)?.label} to continue.` : null}
```

Comment:
We talked about moving this off the radio input and onto the ResponseBlock or ResponseSwitcher. Is that handled by the utils? If so, why is this here?

Reply:
Also, does `&& requiredValue.toString() !== answer.value.toString() ? Please select ${requiredLabel ? requiredLabel : options.find((opt) => opt.value === requiredValue)?.label} to continue. : null` fail to validate if there is no `requiredValue`?

Reply:
And a third thing 🙃 How does this validation work with arrays of required values?

Reply:
This part doesn't really validate, the actual validation is handled entirely in the utils. This is just the UI part that shows the error message in the RadioInput itself. Which is why it doesnt do anything if there is no requiredValue, and why I dont worry about arrays.

Reply:
Gotcha. Should we do this on the other input types too, then?